scripts/patch-realsense-ubuntu-L4T.sh

```diff
@@ -19,10 +19,7 @@ then
 fi
 
 #Include usability functions
-source ./scripts/patch-utils.sh
-
-# Get the required tools to build the patched modules
-sudo apt-get install build-essential git libssl-dev -y
+source ./scripts/patch-utils-hwe.sh
```

Comment:
Will this work on JP4/5?
I thought hwe was introduces at ubuntu 22 no?

Reply:
this backward compatible